src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -192,29 +199,53 @@ export default class CaseAlterationTable extends React.Component<ICaseAlteration
             }
         ];
 
-        const typeSet : {[x: string]: boolean} = {};
+        const geneSet : {[x: string]: OQLLineFilterOutput<AnnotatedExtendedAlteration>} = {};
         this.props.oqls.forEach(oql => {
+            if (oql.gene in geneSet) {
+                geneSet[oql.gene].oql_line += oql.oql_line;
```

Comment:
does the oql_lin value have a delimiter in it so that it can be just concatenated in this way?

Reply:
Yes, the oql_line comes with delimiter at the end.